integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/NextMoveDpiRequest.java

```diff
@@ -0,0 +1,153 @@
+package no.difi.meldingsutveksling.nextmove;
+
+import com.google.common.collect.Lists;
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.domain.MeldingsUtvekslingRuntimeException;
+import no.difi.meldingsutveksling.dpi.Document;
+import no.difi.meldingsutveksling.dpi.MeldingsformidlerRequest;
+import no.difi.meldingsutveksling.serviceregistry.externalmodel.PostAddress;
+import no.difi.meldingsutveksling.serviceregistry.externalmodel.ServiceRecord;
+import org.apache.commons.io.FileUtils;
+import org.springframework.http.MediaType;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public class NextMoveDpiRequest implements MeldingsformidlerRequest {
+
+    private static final String defaultMimeType = MediaType.APPLICATION_PDF_VALUE;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00115)